src/mongocxx/private/index_view.hh

```diff
@@ -134,6 +134,10 @@ class index_view::impl {
             opts_doc.append(bsoncxx::builder::concatenate_doc{session->_get_impl().to_document()});
         }
 
+        if (options.commit_quorum()) {
```

Comment:
I'd suggest a comment above this to explain why you're passing this command-specific option via `opts_bson`, which is typically reserved for generic command options. A reference to [CDRIVER-3748](https://jira.mongodb.org/browse/CDRIVER-3748) would work, like:

> libmongoc validates commitQuorum via command opts (CDRIVER-3748)